cmd/vic-machine/configure/configure.go

```diff
@@ -111,6 +113,12 @@ func (c *Configure) processParams() error {
 		return err
 	}
 
+	var err error
+	usingDHCP := ip.IsUnspecifiedIP(c.Data.ClientNetwork.IP.IP) || ip.IsUnspecifiedIP(c.Data.PublicNetwork.IP.IP) || ip.IsUnspecifiedIP(c.Data.ManagementNetwork.IP.IP)
```

Comment:
I wanted to have this line in common/dns.go, but it uses Data struct in lib/install/data/data.go which already includes the common package so including the Data struct in dns.go would have caused a circular import. Not too sure which function to move this line to. The possible place I see is into data.go itself, but not sure if that's appropriate.